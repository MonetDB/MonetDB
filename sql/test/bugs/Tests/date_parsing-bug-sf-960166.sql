select current_date = date '2004-05-25';
select current_date = '2004-05-25';
