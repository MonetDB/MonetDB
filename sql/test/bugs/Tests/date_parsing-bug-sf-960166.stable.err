stderr of test 'date_parsing-bug-sf-960166` in directory 'sql/test/bugs` itself:


# 09:13:36 >  
# 09:13:36 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/data/Linux-i686/etc/monet.conf" --debug=10 --set "monet_mod_path=/ufs/niels/data/Linux-i686/lib/MonetDB" --set "gdk_dbfarm=/ufs/niels/data/Linux-i686/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/niels/data/Linux-i686/var/MonetDB/log" --set mapi_port=45117 --set sql_port=52668 --set monet_prompt= --trace "--dbname=mTests_src_test_bugs" "/net/stem.ins.cwi.nl/export/scratch1/niels/Linux-i686/build-sql/src/backends/monet4/sql_server.mil" ; echo ; echo Over..
# 09:13:36 >  


# 09:13:39 >  
# 09:13:39 >  Mtimeout -timeout 60 MapiClient --language=sql -u monetdb -P monetdb --host=stem --port=52668  < date_parsing-bug-sf-960166.sql
# 09:13:39 >  


# 09:13:39 >  
# 09:13:39 >  Done.
# 09:13:39 >  

