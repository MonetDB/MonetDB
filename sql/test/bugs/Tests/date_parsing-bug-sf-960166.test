query I rowsort
select current_date = date '2004-05-25'
----
0

query I rowsort
select current_date = '2004-05-25'
----
0

