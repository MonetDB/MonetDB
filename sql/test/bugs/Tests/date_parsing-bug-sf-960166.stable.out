stdout of test 'date_parsing-bug-sf-960166` in directory 'sql/test/bugs` itself:


# 20:18:54 >  
# 20:18:54 >   mserver5 "--config=/ufs/niels/scratch/monetdb/Linux-x86_64/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=4 --set "monet_mod_path=/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/lib:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=34282 --set xrpc_port=47106 --set monet_prompt= --trace  "--dbname=mTests_src_test_bugs" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 20:18:54 >  

# MonetDB server v5.9.0, based on kernel v1.27.0
# Serving database 'mTests_src_test_bugs', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://alf.ins.cwi.nl:34282/
# MonetDB/SQL module v2.27.0 loaded

#function user.main():void;
#    clients.quit();
#end main;


# 11:55:35 >  
# 11:55:35 >  Mtimeout -timeout 60 MapiClient -lsql -umonetdb -Pmonetdb --host=localhost --port=35185  < date_parsing-bug-sf-960166.sql
# 11:55:35 >  

#select current_date = date '2004-05-25';
% .%2 # table_name
% %2 # name
% boolean # type
% 5 # length
[ false	]
#select current_date = '2004-05-25';
% .%2 # table_name
% %2 # name
% boolean # type
% 5 # length
[ false	]

# 12:50:17 >  
# 12:50:17 >  Done.
# 12:50:17 >  

